### DISCUSSION TITLE
Internal behavior of MF6-GWT without the Dispersion package

### DISCUSSION THREAD

**Discussion by reneangermeyer** [modflowpy/flopy#2390] [Q&A]
Hi everyone!

I built an MF6-GWT model using only the Advection package (it converges successfully), and I was wondering what happens internally when the Dispersion package is not included. Does MODFLOW internally implement a simplified version of dispersion, or is there no dispersion at all?

Additionally, what happens to the Courant and Péclet numbers, and how does MODFLOW handle the transport equation in this case? I reviewed the documentation but couldn't find any details regarding this.

Thank you!

**Comment by mbakker7** [comment]
Good question, @reneangermeyer. When you don't include the dispersion package, then MODFLOW doesn't simulate dispersion. However, when MODFLOW simulates advection, it introduces numerical dispersion through the approximation that MODFLOW uses. 
Since you only simulate advection, there is still a Courant number, but no Peclet number.
